lib/json_info_codec.dart

```diff
@@ -7,10 +7,11 @@ part of dart2js_info.info;
 
 // TODO(sigmund): add unit tests.
 class JsonToAllInfoConverter extends Converter<Map<String, dynamic>, AllInfo> {
-  Map<String, Info> registry = <String, Info>{};
+  final Map<String, Info> _registry =
```

Comment:
`registry` is a part of the API for the converter and is used by clients. We can't make it private